AGPs/AGP-1.md

```diff
@@ -65,6 +65,30 @@ Proposals made to the Finance track must affect the movement of assets held by t
   * GPL
   * AGPL
   * MIT
+  
+_Finance track budget_
+
+The total amount funded by Finance track AGPs shall not exceed 5% of the market value of the Association / Aragon Network treasury or 250,000 DAI, whichever is greater, in a given quarterly Aragon Network vote cycle. Recurring transfers approved in previous vote cycles shall be deducted from this budget as described below. The market value of the treasury, and spending relative to that, shall be measured in DAI at the time an attempt is made to process an approved Finance track proposal as described below.
+
+For any Aragon Network vote ballot, approved Finance track proposals will be processed in the following order:
+
+- First, process proposals that reduce or eliminate a previously approved recurring transfer.
```

Comment:
Nitpick: this could be gamed by reducing a recurring proposal by an insignificant amount just to give the proposal maximum processing priority